src/server/home/index.njk

```diff
@@ -45,10 +45,10 @@
 
         <h2 class="govuk-heading-m">Test</h2>
         <p>
-          <a href="/organisations/123/registrations/456" class="govuk-link">Test Registration</a>
+          <a href="{{ langPrefix }}/organisations/123/registrations/456" class="govuk-link">Test Registration</a>
```

Comment:
I wonder if it might be helpful to have/use a `localiseUrl` function, it could be added on a separate ticket/PR but I think it could get tedious doing this everywhere